etc/org.opencastproject.security.aai.ConfigurableLoginHandler.cfg

```diff
@@ -51,3 +54,7 @@
 # Default: ROLE_AAI_USER
 #role.federation = ROLE_AAI_USER
 
+# Role prefix for affiliation
+# Default: ROLE_AAI_USER_AFFILIATION_
+#role.affiliation.prefix = ROLE_AAI_USER_AFFILIATION_
```

Comment:
Maybe add a forth point after line https://github.com/opencast/opencast/pull/812/files#diff-6861a5481b6de717f8e1169f75581425R39

Reply:
Thank you, done.